### DISCUSSION TITLE
Does SS2 support "Socks4" or "Socks5" servers when used as client only?

### DISCUSSION THREAD

**Discussion by MrDkVV** [shadowsocks/go-shadowsocks2#255] [Q&A]
Hello,

I would like to use SS2 as a client only. I use "SSH -D" To establish a socks 4 connection with a remote host of mine. 

I'm really confused as the application suggests it's using "Socks" but I can't find any examples of using the client with a true "Socks Server" (End Point)? 

Please confirm if this is possible and some examples would be awesome ( along with the Socks server used ). 

Thank you in advance.
DK

**Comment by riobard** [comment]
No. This project only provides SOCKS server for browsers to connect.
